### DISCUSSION TITLE
How to use last.fm scrobbler?

### DISCUSSION THREAD

**Discussion by Kingmiami** [pear-devs/pear-desktop#1929] [Q&A] [answered]
I can't find any documentation on how to use the last.fm scrobbler on this application, and it doesn't seem to be as straightforward as it was for the older now defunct Youtube Music for Desktop. Can anyone give a step-by-step?

**Comment by JellyBrick** [comment, marked as answer]
Plugins->scrobblers->last.fm->enable

**Reply by Kingmiami** [reply]
I already had it enabled. It only seemed to work after disabling and re-enabling it, where the sign-up window appeared
